docs/sources/installation/mac.md

```diff
@@ -1,116 +1,306 @@
 
-The Boot2Docker management tool provides several additional commands for working with the
-VM and Docker:
+1. Create a new Boot2Docker VM.
 
-    $ ./boot2docker
-    Usage: ./boot2docker [<options>]
-    {help|init|up|ssh|save|down|poweroff|reset|restart|config|status|info|ip|delete|download|version} [<args>]
+		$ boo2docker init
 
-Continue with the [User Guide](/userguide/).
+	This creates a new virtual machine. You only need to run this command once.
+
+2. Start the `boot2docker` VM. 
+
+		$ boot2docker start
+
+3. Display the environment variables for the Docker client.
+
+		$ boot2docker shellinit
+		Writing /Users/mary/.boot2docker/certs/boot2docker-vm/ca.pem
+		Writing /Users/mary/.boot2docker/certs/boot2docker-vm/cert.pem
+		Writing /Users/mary/.boot2docker/certs/boot2docker-vm/key.pem
+			export DOCKER_HOST=tcp://192.168.59.103:2376
+			export DOCKER_CERT_PATH=/Users/mary/.boot2docker/certs/boot2docker-vm
+
+4. To set the environment variables in your shell do the following:
+
+		$ $(boot2docker shellinit)
+		
+	You can also set them manually by using the `export` commands `boot2docker`
+	returns.
+
+5. Run the `hello-world` container to verify your setup.
+
+		$ docker hello-world		
+
+
+## Basic Boot2Docker Exercises
+
+At this point, you should have `boot2docker` running and the `docker` client
+environment initialized. To verify this, use the following commands:
+
+	$ boot2docker status
+	$ docker version
+			
+Work through this section to try some practical container tasks using `boot2docker` VM.
+
+### Access container ports
+
+1. Start an NGINX container on the DOCKER_HOST.
+
+		$ docker run -d -P --name web nginx
+		
+	Normally, the `docker run` commands starts a container, runs it, and then
+	exits.  The `-d` flag keeps the container running in the background
+	after the `docker` command exits.  The `-P` flag publishes exposed ports from the
+	container to your local host; this lets you access them from your Mac.
+		
+2. Display your running container with `docker ps` command
+
+		CONTAINER ID        IMAGE               COMMAND                CREATED             STATUS              PORTS                                           NAMES
+		5fb65ff765e9        nginx:latest        "nginx -g 'daemon of   3 minutes ago       Up 3 minutes        0.0.0.0:49156->443/tcp, 0.0.0.0:49157->80/tcp   web  
+
+	At this point, you can see `nginx` is running as a daemon. 
+
+3. View just the container's ports.
+
+		$ docker port web
+		443/tcp -> 0.0.0.0:49156
+		80/tcp -> 0.0.0.0:49157
+	
+	This tells you that the `web` container's port `80` is mapped to port
+	`49157` on your Docker host.
+
+4. Enter the `https://localhost:49157` address (`localhost` is `0.0.0.0`) in your browser:
+
+	   ![Bad Address](/installation/images/bad_host.png)
+
+	This didn't work.  The reason it doesn't work is your `DOCKER_HOST` address is
+	not the localhost address (0.0.0.0) but is instead the address of the
```

Comment:
It is a fine balance isn't it?  If you are so new to networking you don't know the term localhost, docker itself is going to tip you into the deep end when you first dip your toe. I actually first introduce the term localhost at the top of the page. I'll drop in a brief definition there.